From f02116f2c5d935e15b4f093b753b9a769f368966 Mon Sep 17 00:00:00 2001
From: Ileana Dumitrescu <ileanadumitrescu95@gmail.com>
Date: Wed, 7 May 2025 17:39:36 +0300
Subject: [PATCH] deplibs-mingw.at: Skip test on MSVC

Test 'deplibs without file command' fails on MSVC with a 'hard failure',
since this test was created for MinGW, skipping for now.

* tests/deplibs-mingw.at: Add check for MSVC build to skip test.
---
 tests/deplibs-mingw.at | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/deplibs-mingw.at b/tests/deplibs-mingw.at
index bfa7b25d..1d997ff2 100644
--- a/tests/deplibs-mingw.at
+++ b/tests/deplibs-mingw.at
@@ -24,6 +24,11 @@
 AT_SETUP([deplibs without file command])
 AT_KEYWORDS([libtool])
 
+# Skip test for MSVC
+if $LIBTOOL --config | $EGREP '^nm_interface="MS dumpbin"' >/dev/null; then
+  AT_CHECK([exit 77])
+fi
+
 cwd=`pwd`
 instdir=$cwd/inst
 libdir=$instdir/lib
-- 
2.45.2

